modelgym/gp_trainer.py

```diff
@@ -43,7 +43,7 @@ def crossval_fit_eval(self, model, cv_pairs, params=None, n_estimators=None, ver
         params = params or model.default_params
         n_estimators = n_estimators or self.n_estimators
         if (isinstance(params, list)):
-            model.set_parameter(params)
+            model.set_parameters(params)
```

Comment:
don't you need to `preprocess_params` here as well?

Reply:
not really, cos it calls `super` method where `preprocess_params` is runned